lib/file.js

```diff
@@ -218,12 +218,13 @@ class SourceFile extends File {
      * @param {import('./typedefs').visitor.ParamInfo[]} parameters - list of parameters, passed as [name, modifier, type] tuple
      * @param {string} returns - the return type of the function
      * @param {'function' | 'action'} kind - kind of the node
-     * @param {string[]?} doc - documentation for the function
+     * @param {?string[]} doc - documentation for the function
```

Comment:
I am unfamiliar with the leading `?` in parameter types. Does that have special semantics or is it a typo?

Reply:
typo :)

Reply:
Fixed